Annex60/Fluid/BaseClasses/PartialThreeWayResistance.mo

```diff
@@ -78,87 +76,44 @@ partial model PartialThreeWayResistance
     final X_start=X_start,
     final C_start=C_start,
     final allowFlowReversal=true,
-    final prescribedHeatFlowRate=false) if
-       dynamicBalance "Fluid volume to break algebraic loop"
+    final prescribedHeatFlowRate=false) "Fluid volume to break algebraic loop"
     annotation (Placement(transformation(extent={{-10,0},{10,20}})));
 
-protected
```

Comment:
@rubenbaetens : The instance `port_internal` was added by @Mathadon  in https://github.com/iea-annex60/modelica-annex60/commit/bdc294e88138deebb072cb952f54378955c0cd84 I think he wanted this to have simpler equations.
@Mathadon : Is this correct? [If yes, there should be a unit test that gives different results after this change.]